exasol_integration_test_docker_environment/test/test_common_run_task_subprocess.py

```diff
@@ -69,12 +69,34 @@ def run_test_different_logging_file_raises_error() -> None:
     with tempfile.TemporaryDirectory() as temp_dir:
         task_creator = lambda: generate_root_task(task_class=TestTask, x="Test")
 
-        success, task = run_task(task_creator, workers=5, task_dependencies_dot_file=None)
-        assert success
+        run_task(task_creator, workers=5, task_dependencies_dot_file=None)
         log_path = Path(temp_dir) / "main.log"
         os.environ[LOG_ENV_VARIABLE_NAME] = str(log_path)
-        success, task = run_task(task_creator, workers=5, task_dependencies_dot_file=None)
-        assert success is False
+        exception_thrown = False
+        try:
+            run_task(task_creator, workers=5, task_dependencies_dot_file=None)
+        except ValueError as e:
+            exception_thrown = True
+        assert exception_thrown
```

Comment:
This here is a bit tricky. I need to run all these tests in a different process, due to luigi's logging limitations. However, I can't use UnitTests.TestCase then, as it would be collected as normal test. Because of this I don't use TestCase.assertXXX here